.github/workflows/daily.yml

```diff
@@ -0,0 +1,39 @@
+name: ci
+
+on:
+  schedule:
+    - cron: "0 0 * * *"
+
+jobs:
+  daily:
+    runs-on: ubuntu-latest
+    if: contains(github.ref, 'refs/heads/master')
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v2
+
+      - name: Set up Go
+        uses: actions/setup-go@v2
+        with:
+          go-version: 1.16
+
+      - name: Generate top 100
+        run: go run . -count 100 > top-100.tsv
+
+      - name: Generate top 1000
+        run: go run . -count 1000 > top-1000.tsv
+
+      - name: Generate top 10000
+        run: go run . -count 10000 > top-10000.tsv
+
+      - name: Daily update
+        uses: drud/action-cross-commit@master
+        with:
+          source-folder: .
+          destination-repository: https://mvdan:${{ secrets.GITHUB_TOKEN }}@github.com/mvdan/corpus
+          destination-folder: .
+          destination-branch: master
+          git-user: auto-updater
+          git-user-email: auto-updater@mvdan-corpus.com
```

Comment:
Git user and user-email can be configured in appropriate values. Note that these don't need to be actual valid username and emails.

Reply:
how about `auto-updater@users.noreply.github.com`?